pkg/migrate/migrate.go

```diff
@@ -1146,3 +1296,249 @@ func resetReclaimPolicy(ctx context.Context, w *log.Logger, clientset k8sclient.
 
 	return nil
 }
+
+// buildPVConsumerPod creates a pod spec for consuming a pvc
+func (p *PVMigrator) buildPVConsumerPod(pvcName string) *corev1.Pod {
+	podName := p.podNameOverride
+	if podName == "" {
+		tmp := uuid.New().String()[:5]
+		podName = fmt.Sprintf("pvmigrate-vol-consumer-%s-%s", pvcName, tmp)
+	}
+	if len(podName) > 63 {
+		podName = podName[0:31] + podName[len(podName)-32:]
+	}
+	return &corev1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      podName,
+			Namespace: "default",
+		},
+		Spec: corev1.PodSpec{
+			RestartPolicy: corev1.RestartPolicyNever,
+			Volumes: []corev1.Volume{
+				{
+					Name: "tmp",
+					VolumeSource: corev1.VolumeSource{
+						PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
+							ClaimName: pvcName,
+						},
+					},
+				},
+			},
+			Containers: []corev1.Container{
+				{
+					Name:  "busybox",
+					Image: "busybox",
```

Comment:
Will this work for airgap customers?

Reply:
Good point! This image would need to be loaded into Docker or containerd unless it's been loaded already by another image which builds atop of `busybox`. 

Hmm, @ricardomaraschini node disk space validator would also run into the same issue if it was integrated in the kURL installer. The disk space validator relies on the [`rsync`](https://github.com/replicatedhq/kURL/blob/main/kurl_util/cmd/pvmigrate/main.go#L34) image to be available. It's probably a good idea for both to use the same container image. I need to think about how to best handle this. I'm currently thinking we should pick an image to use, archive it and then load it during install.

Reply:
We already require an image to be passed via the CLI to do the migration with
Use that image

Reply:
https://github.com/replicatedhq/pvmigrate/blob/075f71c081bc4cdca2575e9ad30ea9ee0251a107/pkg/migrate/migrate.go#L83